### PR TITLE
Portuguese Translation (v3.8.1)

### PR DESCRIPTION
**Pull request by Frediy99**
ING: ZalfredomZ
Discord: Freed#5762

### CODE REVIEW

**Comment by mty22** [comment]
thank you! will look into this one asap!

**Comment by mty22** [comment]
Need a native speaker to review.

**Comment by Frediy99** [comment]
@mty22 Hey my friend, in this translation, i did more than 500 edited lines, so i´ll get 50.000 kr and verified, if a revier approve my translation?

**Comment by mty22** [comment]
> @mty22 Hey my friend, in this translation, i did more than 500 edited lines, so i´ll get 50.000 kr and verified, if a revier approve my translation? (And yes, i have sure that i did more than 500, i counted one by one)

Hi @Frediy99 -- https://github.com/mty22/krunker.io-translations/blob/main/kr_payment_guide.txt#L20

50,000 KR + Verification is typically reserved for new languages, although exceptions can be made its just a "KR payment guide". 

Once the review is done (and depending on the quality of the work), you'll be rewarded accordingly. :) 

Also, the reviewer will also be taking a small percentage of the reward (since it does take quite a bit of time to review).

File: pr.js [hunk 1]
```diff
@@ -1,777 +1,777 @@
-module.exports = `
-menu.btn.ranked Ranked
```
Comment (CRAZYGIRLS22) [inline pr.js:2]: Classificação

File: pr.js [hunk 2]
```diff
@@ -1,777 +1,777 @@
...
+settings.hitmarker.kcolor Cor de Abate
+settings.hitmarker.thickness Espessura
+settings.hitmarker.opacity Opacidade
+settings.hitmarker.leng Comprimento
+settings.hitmarker.spacing Espaçamento
+settings.hitmarker.animD Tamanho de Abrimento
+settings.hitmarker.animS Velocidade de Abrimento
+settings.hitmarker.hitm Mostrar
+settings.hitmarker.hitmFad Velocidade de Transição
+settings.hitmarker.custom Imagem do HitMarker
```
Comment (CRAZYGIRLS22) [inline pr.js:237]: Imagem Do Marcador

File: pr.js [hunk 3]
```diff
@@ -1,777 +1,777 @@
...
+settings.hitmarker.custom.paste URL da imagem do Hitmarker
```
Comment (CRAZYGIRLS22) [inline pr.js:238]: URL Da Imagem Do Marcador

File: pr.js [hunk 4]
```diff
@@ -1,777 +1,777 @@
...
+settings.crosshair.color Cor
+settings.crosshair.shadow.color Cor da Sombra
+settings.crosshair.shadow.thickness Espessura da Sombra
+settings.crosshair.size Tamanho
+settings.crosshair.thickness Espessura
+settings.crosshair.gap Espaçamento
+settings.crosshair.dot Meio
+settings.crosshair.opacity.use Usar Opacidade Customizada
+settings.crosshair.opacity.custom Opacidade Customizada
+settings.hitmarker.header Marcador de Hit
```
Comment (CRAZYGIRLS22) [inline pr.js:226]: Marcador De Golpe

File: pr.js [hunk 5]
```diff
@@ -1,777 +1,777 @@
...
+settings.interface.hideNames.all Todo Mundo
+settings.interface.nametagStyle Tipo de Barra de Vida
+settings.interface.nametagOpac Opacidade da Barra de vida
+settings.interface.nametagStyle.health Apenas Vida
+settings.interface.nametagStyle.name Apenas Nome
+settings.interface.nametagStyle.nameLvl Nome e Level Apenas
+settings.interface.nametagStyle.all Tudo
+settings.interface.healthNum Número da Barra de Vida
+settings.interface.healthColT Cor da Barra de Vida do Time
+settings.interface.healthColE Cor da barrra de Vida Inimiga
```
Comment (CRAZYGIRLS22) [inline pr.js:199]: Cor Da Barra De Vida Do Inimigo

File: pr.js [hunk 6]
```diff
@@ -1,777 +1,777 @@
...
+settings.interface.hideNames.enemy Apenas Inimigos
+settings.interface.hideNames.all Todo Mundo
+settings.interface.nametagStyle Tipo de Barra de Vida
+settings.interface.nametagOpac Opacidade da Barra de vida
+settings.interface.nametagStyle.health Apenas Vida
+settings.interface.nametagStyle.name Apenas Nome
+settings.interface.nametagStyle.nameLvl Nome e Level Apenas
+settings.interface.nametagStyle.all Tudo
+settings.interface.healthNum Número da Barra de Vida
+settings.interface.healthColT Cor da Barra de Vida do Time
```
Comment (CRAZYGIRLS22) [inline pr.js:198]: Cor Da Barra De Vida Da Minha Equipa

File: pr.js [hunk 7]
```diff
@@ -1,777 +1,777 @@
...
+settings.interface.oldBrowser Usar o antigo Navegador
+settings.interface.rarityanim Desativar animações raras
+settings.interface.showPfp Mostar foto de Perfil
+settings.interface.showTradeMsg Mostrar Mensagens de Trade
+settings.interface.hp Barra Dinâmica de Vida
+settings.interface.dmgScale Escala de Dano
+settings.interface.killf Mostrar mensagem de abate
+settings.interface.ping Mostrar Qualidade da Conexão
+settings.interface.net-stats Mostar status de conexão
+settings.interface.fps Mostrar Frames Por Segundo
```
Comment (CRAZYGIRLS22) [inline pr.js:172]: Quadros Por Segundo

File: pr.js [hunk 8]
```diff
@@ -1,777 +1,777 @@
...
+settings.interface.dmgCol Cor do dano
+settings.interface.critCol Cor do Crítico
+settings.interface.header Interface
+settings.interface.scale Scala da Tela
+settings.interface.ui Mostrar Interface de Usuário
+settings.interface.scoreboard Usar o antigo Placar de Pontos
+settings.interface.oldBrowser Usar o antigo Navegador
+settings.interface.rarityanim Desativar animações raras
+settings.interface.showPfp Mostar foto de Perfil
+settings.interface.showTradeMsg Mostrar Mensagens de Trade
```
Comment (CRAZYGIRLS22) [inline pr.js:166]: Mostrar Mensagens Das Trocas

File: pr.js [hunk 9]
```diff
@@ -1,777 +1,777 @@
...
+settings.post.bloomRadius Área do Contraste
+settings.post.ssao Intencificardo da Iluminação
+settings.post.ssaoRadius Área de Intencificação da Iluminação
+settings.interface.hitInd Mostrar indicador de Hit
+settings.interface.hitColor Cor do indicardor de hit
+settings.interface.dmg Mostrar Dano
+settings.interface.dmgCol Cor do dano
+settings.interface.critCol Cor do Crítico
+settings.interface.header Interface
+settings.interface.scale Scala da Tela
```
Comment (CRAZYGIRLS22) [inline pr.js:160]: Escala Do Ecrâ

File: pr.js [hunk 10]
```diff
@@ -1,777 +1,777 @@
...
+settings.post.header Pós-Processamento
+settings.post.post Pós-Processamento
+settings.post.bloom Contraste
+settings.post.bloomThreshold Compactação do Contraste
+settings.post.bloomStrength Força do Contraste
+settings.post.bloomRadius Área do Contraste
+settings.post.ssao Intencificardo da Iluminação
+settings.post.ssaoRadius Área de Intencificação da Iluminação
+settings.interface.hitInd Mostrar indicador de Hit
+settings.interface.hitColor Cor do indicardor de hit
```
Comment (CRAZYGIRLS22) [inline pr.js:155]: Cor De Indicador De Golpe

File: pr.js [hunk 11]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.aspectRatio Proporção de Tela
+settings.post.header Pós-Processamento
+settings.post.post Pós-Processamento
+settings.post.bloom Contraste
+settings.post.bloomThreshold Compactação do Contraste
+settings.post.bloomStrength Força do Contraste
+settings.post.bloomRadius Área do Contraste
+settings.post.ssao Intencificardo da Iluminação
+settings.post.ssaoRadius Área de Intencificação da Iluminação
+settings.interface.hitInd Mostrar indicador de Hit
```
Comment (CRAZYGIRLS22) [inline pr.js:154]: Mostrar Indicador De Golpe

File: pr.js [hunk 12]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.ambient Sombriamento do Ambiente
+settings.quality.ambientOld Antigo Sombriamento (Quando Possível)
+settings.quality.textureAnim Animação de Texturas
+settings.quality.objectAnim Animação de Objetos
+settings.quality.screenShake Tremor de tela
+settings.quality.weaponShine Brilho na Arma
+settings.quality.rendGamma Gamma
+settings.quality.lighting Iluminação
+settings.quality.showExplo Mostrar Explosão
+settings.quality.aspectRatio Proporção de Tela
```
Comment (CRAZYGIRLS22) [inline pr.js:145]: Proporção Do Ecrâ

File: pr.js [hunk 13]
```diff
@@ -1,777 +1,777 @@
...
+settings.post.header Pós-Processamento
+settings.post.post Pós-Processamento
+settings.post.bloom Contraste
+settings.post.bloomThreshold Compactação do Contraste
+settings.post.bloomStrength Força do Contraste
+settings.post.bloomRadius Área do Contraste
+settings.post.ssao Intencificardo da Iluminação
```
Comment (CRAZYGIRLS22) [inline pr.js:152]: Intensidade Da Iluminação

File: pr.js [hunk 14]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.trails Rastro de Balas
+settings.quality.yourTrails Seus Ratros
+settings.quality.mflash Trilha dos Tiros
+settings.quality.flap Flap da Sniper
+settings.quality.lowSpec Personagens Sem Pernas
+settings.quality.ambient Sombriamento do Ambiente
+settings.quality.ambientOld Antigo Sombriamento (Quando Possível)
+settings.quality.textureAnim Animação de Texturas
+settings.quality.objectAnim Animação de Objetos
+settings.quality.screenShake Tremor de tela
```
Comment (CRAZYGIRLS22) [inline pr.js:140]: Ecrâ Tremer

File: pr.js [hunk 15]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.softShad Sombras Suaves
+settings.quality.highResShad Sombras de Alta Resolução
+settings.quality.shadowsDynamic Sombras Dinâmicas
+settings.quality.trails Rastro de Balas
+settings.quality.yourTrails Seus Ratros
+settings.quality.mflash Trilha dos Tiros
+settings.quality.flap Flap da Sniper
+settings.quality.lowSpec Personagens Sem Pernas
+settings.quality.ambient Sombriamento do Ambiente
+settings.quality.ambientOld Antigo Sombriamento (Quando Possível)
```
Comment (CRAZYGIRLS22) [inline pr.js:137]: Antigo Sombreado (Quando Possível)

File: pr.js [hunk 16]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.shadows Sombras
+settings.quality.softShad Sombras Suaves
+settings.quality.highResShad Sombras de Alta Resolução
+settings.quality.shadowsDynamic Sombras Dinâmicas
+settings.quality.trails Rastro de Balas
+settings.quality.yourTrails Seus Ratros
+settings.quality.mflash Trilha dos Tiros
+settings.quality.flap Flap da Sniper
+settings.quality.lowSpec Personagens Sem Pernas
+settings.quality.ambient Sombriamento do Ambiente
```
Comment (CRAZYGIRLS22) [inline pr.js:136]: Sombreado Do Ambiente

File: pr.js [hunk 17]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.renderdist Distância de Renderização
+settings.quality.reflection Qualidade de Reflexço 
+settings.quality.antiAlias Anti-Serrilhamento
+settings.quality.shadows Sombras
+settings.quality.softShad Sombras Suaves
+settings.quality.highResShad Sombras de Alta Resolução
+settings.quality.shadowsDynamic Sombras Dinâmicas
+settings.quality.trails Rastro de Balas
+settings.quality.yourTrails Seus Ratros
+settings.quality.mflash Trilha dos Tiros
```
Comment (CRAZYGIRLS22) [inline pr.js:133]: Rasto

File: pr.js [hunk 18]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.partdist Distãncia de Particulas
+settings.quality.renderdist Distância de Renderização
+settings.quality.reflection Qualidade de Reflexço 
+settings.quality.antiAlias Anti-Serrilhamento
+settings.quality.shadows Sombras
+settings.quality.softShad Sombras Suaves
+settings.quality.highResShad Sombras de Alta Resolução
+settings.quality.shadowsDynamic Sombras Dinâmicas
+settings.quality.trails Rastro de Balas
+settings.quality.yourTrails Seus Ratros
```
Comment (CRAZYGIRLS22) [inline pr.js:132]: Meu Rasto

File: pr.js [hunk 19]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.noTex Sem Texturas
+settings.quality.partdist Distãncia de Particulas
+settings.quality.renderdist Distância de Renderização
+settings.quality.reflection Qualidade de Reflexço 
+settings.quality.antiAlias Anti-Serrilhamento
+settings.quality.shadows Sombras
+settings.quality.softShad Sombras Suaves
+settings.quality.highResShad Sombras de Alta Resolução
+settings.quality.shadowsDynamic Sombras Dinâmicas
+settings.quality.trails Rastro de Balas
```
Comment (CRAZYGIRLS22) [inline pr.js:131]: Rasto de

File: pr.js [hunk 20]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.updRate Limitação de Frames
+settings.quality.rendDst Distância de Renderização
+settings.quality.res Resolução
+settings.quality.part Partículas
+settings.quality.mapDet Detalhes de Mapas 
+settings.quality.noTex Sem Texturas
+settings.quality.partdist Distãncia de Particulas
+settings.quality.renderdist Distância de Renderização
+settings.quality.reflection Qualidade de Reflexço 
+settings.quality.antiAlias Anti-Serrilhamento
```
Comment (CRAZYGIRLS22) [inline pr.js:126]: Anti-Desfocado

File: pr.js [hunk 21]
```diff
@@ -1,777 +1,777 @@
...
+settings.quality.header Qualidade
+settings.quality.updRate Limitação de Frames
+settings.quality.rendDst Distância de Renderização
+settings.quality.res Resolução
+settings.quality.part Partículas
+settings.quality.mapDet Detalhes de Mapas 
+settings.quality.noTex Sem Texturas
+settings.quality.partdist Distãncia de Particulas
+settings.quality.renderdist Distância de Renderização
+settings.quality.reflection Qualidade de Reflexço
```
Comment (CRAZYGIRLS22) [inline pr.js:125]: Qualidade De Reflexo

File: pr.js [hunk 22]
```diff
@@ -1,777 +1,777 @@
...
+settings.chat.chatTextOutline Contorno de Texto do Chat
+settings.chat.chatFilter Fitro de palavrões
+settings.quality.header Qualidade
+settings.quality.updRate Limitação de Frames
+settings.quality.rendDst Distância de Renderização
+settings.quality.res Resolução
+settings.quality.part Partículas
+settings.quality.mapDet Detalhes de Mapas 
+settings.quality.noTex Sem Texturas
+settings.quality.partdist Distãncia de Particulas
```
Comment (CRAZYGIRLS22) [inline pr.js:123]: Distância De Partículas

File: pr.js [hunk 23]
```diff
@@ -1,777 +1,777 @@
...
+settings.chat.messages Mostrar Mensagem de Players
+settings.chat.unboxings Mostrar Unoxing de Player
+settings.chat.chatOp Opacidade de Chat
+settings.chat.chatBGOp Opacidade do Fundo do Chat
+settings.chat.chatHeight Altura do Chat
+settings.chat.chatTimer Tempo do Chat (s)
+settings.chat.chatTextOutline Contorno de Texto do Chat
+settings.chat.chatFilter Fitro de palavrões
+settings.quality.header Qualidade
+settings.quality.updRate Limitação de Frames
```
Comment (CRAZYGIRLS22) [inline pr.js:117]: Limitação De Quadros

File: pr.js [hunk 24]
```diff
@@ -1,777 +1,777 @@
...
+settings.chat.chat.all Sempre
+settings.chat.chat.focus Sempre Focado
+settings.chat.messages Mostrar Mensagem de Players
+settings.chat.unboxings Mostrar Unoxing de Player
+settings.chat.chatOp Opacidade de Chat
+settings.chat.chatBGOp Opacidade do Fundo do Chat
+settings.chat.chatHeight Altura do Chat
+settings.chat.chatTimer Tempo do Chat (s)
+settings.chat.chatTextOutline Contorno de Texto do Chat
+settings.chat.chatFilter Fitro de palavrões
```
Comment (CRAZYGIRLS22) [inline pr.js:115]: Filtro De Mensagens

File: pr.js [hunk 25]
```diff
@@ -1,777 +1,777 @@
...
+settings.experimental.mouseFlickFix Correção de Flick de Mouse
+settings.experimental.shaderRendering Renderização Intença 
+settings.dev.header Desenvolvidor
+settings.dev.forceUpdate Forçar atualização de skins
+settings.dev.forceRegen Sempre Regenerar Players
+settings.dev.extraBan Informação De Ban Extra
+settings.chat.header Chat
+settings.chat.chat Sempre mostrar O Chat
+settings.chat.chat.all Sempre
+settings.chat.chat.focus Sempre Focado
```
Comment (CRAZYGIRLS22) [inline pr.js:107]: Foco Das Mensagens

File: pr.js [hunk 26]
```diff
@@ -1,777 +1,777 @@
...
+settings.experimental.header Configurações Experimentais
+settings.experimental.aimFreezeFix Correção De Mira Parada
+settings.experimental.mouseFlickFix Correção de Flick de Mouse
+settings.experimental.shaderRendering Renderização Intença 
+settings.dev.header Desenvolvidor
+settings.dev.forceUpdate Forçar atualização de skins
+settings.dev.forceRegen Sempre Regenerar Players
+settings.dev.extraBan Informação De Ban Extra
+settings.chat.header Chat
+settings.chat.chat Sempre mostrar O Chat
```
Comment (CRAZYGIRLS22) [inline pr.js:105]: Mostrar Sempre As Mensagens

File: pr.js [hunk 27]
```diff
@@ -1,777 +1,777 @@
...
+settings.chat.chat.all Sempre
+settings.chat.chat.focus Sempre Focado
+settings.chat.messages Mostrar Mensagem de Players
```
Comment (CRAZYGIRLS22) [inline pr.js:108]: Mostrar Mensagem De Jogadores

File: pr.js [hunk 28]
```diff
@@ -1,777 +1,777 @@
...
+settings.experimental.aimFreezeFix Correção De Mira Parada
+settings.experimental.mouseFlickFix Correção de Flick de Mouse
+settings.experimental.shaderRendering Renderização Intença 
+settings.dev.header Desenvolvidor
+settings.dev.forceUpdate Forçar atualização de skins
+settings.dev.forceRegen Sempre Regenerar Players
+settings.dev.extraBan Informação De Ban Extra
+settings.chat.header Chat
+settings.chat.chat Sempre mostrar O Chat
+settings.chat.chat.all Sempre
```
Comment (CRAZYGIRLS22) [inline pr.js:106]: Motrar Todas As Mensagens

File: pr.js [hunk 29]
```diff
@@ -1,777 +1,777 @@
...
+settings.network.netRerouting Rotação Automática (Afeta A Conexão)
+settings.network.lagComp Compensação de Lag
+settings.popup.header Pop-Ups
+settings.popup.freeKRPop Mostrar Pop-Up de KR
+settings.experimental.header Configurações Experimentais
+settings.experimental.aimFreezeFix Correção De Mira Parada
+settings.experimental.mouseFlickFix Correção de Flick de Mouse
+settings.experimental.shaderRendering Renderização Intença 
+settings.dev.header Desenvolvidor
+settings.dev.forceUpdate Forçar atualização de skins
```
Comment (CRAZYGIRLS22) [inline pr.js:101]: Forçar Atualização De Peles

File: pr.js [hunk 30]
```diff
@@ -1,777 +1,777 @@
...
+settings.dev.forceRegen Sempre Regenerar Players
```
Comment (CRAZYGIRLS22) [inline pr.js:102]: Forçar Reviver De Jogador

File: pr.js [hunk 31]
```diff
@@ -1,777 +1,777 @@
...
+settings.network.sendRate Tickrate Alto
+settings.network.netRerouting Rotação Automática (Afeta A Conexão)
+settings.network.lagComp Compensação de Lag
+settings.popup.header Pop-Ups
+settings.popup.freeKRPop Mostrar Pop-Up de KR
+settings.experimental.header Configurações Experimentais
+settings.experimental.aimFreezeFix Correção De Mira Parada
+settings.experimental.mouseFlickFix Correção de Flick de Mouse
+settings.experimental.shaderRendering Renderização Intença 
+settings.dev.header Desenvolvidor
```
Comment (CRAZYGIRLS22) [inline pr.js:100]: Cabeçalho de Desenvolvedor

File: pr.js [hunk 32]
```diff
@@ -1,777 +1,777 @@
...
+settings.network.header Network
+settings.network.sendRate Tickrate Alto
+settings.network.netRerouting Rotação Automática (Afeta A Conexão)
+settings.network.lagComp Compensação de Lag
+settings.popup.header Pop-Ups
+settings.popup.freeKRPop Mostrar Pop-Up de KR
+settings.experimental.header Configurações Experimentais
+settings.experimental.aimFreezeFix Correção De Mira Parada
+settings.experimental.mouseFlickFix Correção de Flick de Mouse
+settings.experimental.shaderRendering Renderização Intença
```
Comment (CRAZYGIRLS22) [inline pr.js:99]: Detalhe De Renderização Intensa

File: pr.js [hunk 33]
```diff
@@ -1,777 +1,777 @@
...
+player.disconnect DESCONECTADO
+timer.end Próxima Partida Em {0}
+leaderboard.empty Sala Vazia
+popup.checkpoint Ponto de Renascimento
+app.play CLIQUE PARA JOGAR
+app.spectate CLIQUE PARA ESPECTAR
+app.play.mobile TOQUE PARA JOGAR
+app.spectate.mobile TOQUE PARA ESPECTAR
+app.play.controller PRESSIONE COMEÇAR
+app.spectate.controller START PARA ESPECTAR
```
Comment (CRAZYGIRLS22) [inline pr.js:75]: Comece A Espectar O Jogo

File: pr.js [hunk 34]
```diff
@@ -1,777 +1,777 @@
...
+player.died Você Foi Abatido
+player.killed Abatido Por
+player.killstreak {0} Em Uma {1} Sequência de Abates
+player.inactive Expulso por Inatividade
+player.disconnect DESCONECTADO
+timer.end Próxima Partida Em {0}
+leaderboard.empty Sala Vazia
+popup.checkpoint Ponto de Renascimento
+app.play CLIQUE PARA JOGAR
+app.spectate CLIQUE PARA ESPECTAR
```
Comment (CRAZYGIRLS22) [inline pr.js:71]: Clique Para Espectar Jogo

File: pr.js [hunk 35]
```diff
@@ -1,777 +1,777 @@
...
+app.play.mobile TOQUE PARA JOGAR
+app.spectate.mobile TOQUE PARA ESPECTAR
```
Comment (CRAZYGIRLS22) [inline pr.js:73]: Toque Para Espectar Jogadores

File: pr.js [hunk 36]
```diff
@@ -1,777 +1,777 @@
...
+player.killed Abatido Por
+player.killstreak {0} Em Uma {1} Sequência de Abates
+player.inactive Expulso por Inatividade
+player.disconnect DESCONECTADO
+timer.end Próxima Partida Em {0}
+leaderboard.empty Sala Vazia
+popup.checkpoint Ponto de Renascimento
+app.play CLIQUE PARA JOGAR
+app.spectate CLIQUE PARA ESPECTAR
+app.play.mobile TOQUE PARA JOGAR
```
Comment (CRAZYGIRLS22) [inline pr.js:72]: Toque Para Jogar

File: pr.js [hunk 37]
```diff
@@ -1,777 +1,777 @@
...
+generic.accuracy Precisão
+player.waiting Esperando Por Jogadores
+player.died Você Foi Abatido
+player.killed Abatido Por
+player.killstreak {0} Em Uma {1} Sequência de Abates
+player.inactive Expulso por Inatividade
+player.disconnect DESCONECTADO
+timer.end Próxima Partida Em {0}
+leaderboard.empty Sala Vazia
+popup.checkpoint Ponto de Renascimento
```
Comment (CRAZYGIRLS22) [inline pr.js:69]: Ponto De Verificação

File: pr.js [hunk 38]
```diff
@@ -1,777 +1,777 @@
...
+generic.unfollow Deixar de Seguir
+generic.followers Seguidores
+generic.following Seguindo
+generic.games Jogos Da Comunidade
+generic.wins Vitórias
+generic.lose Derrotras
+generic.clan Clã
+generic.games.played Partidas Jogadas
+generic.games.won Partidas Ganhas
+generic.games.lost Partidas Derrotadas
```
Comment (CRAZYGIRLS22) [inline pr.js:48]: Derrotas

File: pr.js [hunk 39]
```diff
@@ -1,777 +1,777 @@
...
-item.view.open Open Model Viewer
-item.view.more More Information
-item.sell.cant Item cant be sold.
-item.sell.quick Are you sure you want to Quick Sell {0}?
-social.profile.stats Overall Stats
-social.profile.class Class Stats
-social.profile.sales Sales
-social.profile.sales.market Market Sales
+module.exports = `
+menu.btn.ranked Ranqueada
```
Comment (CRAZYGIRLS22) [inline pr.js:2]: Classificada

**Comment by mty22** [comment]
@CRAZYGIRLS22 thank you so much for the detailed review! please leave a comment with your IGN and Discord user.

@Frediy99 can you check @CRAZYGIRLS22's review, and fix up anything required? Please click "Resolve conversation" so I know what's complete and what's not complete!

**Comment by CRAZYGIRLS22** [comment]
Ign : CRAZYGIRLS22
DiscordNameTag : CRAZYGIRLS22  #5983

Always a pleasure to help!

**Comment by Frediy99** [comment]
@CRAZYGIRLS22 Bro wtf, this wrongs u checked i already translated, u prob are looking the wrong file, bruh, if u go in "file changed", there are the entire translation, and two columns, one red, with the orginal file, an other green, my full translation @mty22

**Comment by CRAZYGIRLS22** [comment]
My bad, yeah I asked in krunker bunker about it, someone told me I did it
in wrong place. Ima take a look when arrive home.
I want to help, I checked the website were it "teaches" how to do it, and
still going to wrong one.
But another person told me to open that one and edit that one, so it looked
kinda confused for a moment.
Promise I'll try my best next time.
Thank you for telling me. Have a great day

A quarta, 28/04/2021, 12:51, Frediy99 ***@***.***> escreveu:

**Comment by Frediy99** [comment]
@CRAZYGIRLS22 No problem my friend, wrongs happen, but try to review my translation in "file changed", and the text need be "green" , so will be my translation
I´m beliving in you, do your better (;

**Review by CRAZYGIRLS22 — APPROVED** [review]

**Review by CRAZYGIRLS22 — APPROVED** [review]

**Review by CRAZYGIRLS22 — APPROVED** [review]

**Comment by mty22** [comment]
Good to merge? @Frediy99 @CRAZYGIRLS22

**Comment by Frediy99** [comment]
@mty22 Yes bro, the rewards, 90% its for me, 10% its for @CRAZYGIRLS22, bc he helped my a little

**Comment by CRAZYGIRLS22** [comment]
I think its all  ready to update in-game.
I agree and was a pleasure to help :)

**Comment by mty22** [comment]
> @mty22 Yes bro, the rewards, 90% its for me, 10% its for @CRAZYGIRLS22, bc he helped my a little

@CRAZYGIRLS22 do you agree with this?

also what's your Discord/IGN @CRAZYGIRLS22 

---
ING: ZalfredomZ
Discord: Freed#5762
---

**Comment by CRAZYGIRLS22** [comment]
Yes i do!
IGN:  CRAZYGIRLS22
C O P S P 1 1 R E [WHYT] #5983

**Comment by mty22** [comment]
KR queued

**Comment by mty22** [comment]
kb roles sorted

**Pull request merged**